lib/pages/timetable/view/events/event_view.dart

```diff
@@ -85,21 +90,18 @@ class _EventViewState extends State<EventView> {
                     crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
                       Text(
-                          widget.eventInstance.title ??
-                              widget.eventInstance.mainEvent.type
-                                  .toLocalizedString(),
+                          widget.eventInstance?.title ??
+                              mainEvent.type.toLocalizedString(),
                           style: Theme.of(context).textTheme.headline6),
                       const SizedBox(height: 4),
-                      Text(widget.eventInstance.dateString),
-                      if (widget.eventInstance.mainEvent is RecurringUniEvent &&
+                      if (widget.eventInstance != null)
+                        Text(widget.eventInstance.dateString),
+                      if (mainEvent is RecurringUniEvent &&
                           LocaleProvider.rruleL10n != null)
```

Comment:
I'd rather not duplicate the logic in `info` here, since it will change soon and that will lead to bugs. I'm not sure what the best approach would be here, how about `if (mainEvent.info != widget.eventInstance.dateString)`? So to say "as long as we're not duplicating the row above, let's show whatever it is in the `info` field".

WDYT?

Reply:
Tried it and it works consistently, I agree with you